views/_includes/input-dates.pug

```diff
@@ -0,0 +1,24 @@
+mixin dateInputs(idPrefix, legendText, isoHintText, values)
+  div.date-container(class={['has-error']: errors && errors[`${idPrefix}Day`]})
+    fieldset.fieldset(role="group", aria-describedby=`${idPrefix}Hint`,)
+      legend
+        h2 #{__(legendText)}
+
+      span#dobHint.cra-form-message #{__('For Example:')} #{isoDateHintText(isoHintText)}
+
+      if errors
+        ul
+          each err in errors
+            li
+              +validationMessage(err.msg, err.param)
```

Comment:
This starts working as soon as there are errors, but we have some pages where the date part is only part of the errors that are possible.

Reply:
https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/views/login/questions/child.pug#L39